### DISCUSSION TITLE
Unable to get approved for Github Developer student pack

### DISCUSSION THREAD

**Discussion by sarthak263** [community/community#142910] [GitHub Education]
### Hi everyone,

Currently, I am doing MS in Artificial Intelligence at John Hopkins, which is an Online program for students. I recently applied for student developer pack, but I keep getting unable to verify academic affiliation. I provided my unofficial transcript as proof and for online enrollment program I provided my current class with address that the location is Online. It might be because I can only use front facing camera instead of my back camera, so every picture I take is very blurry. Is there another way where I can get approved. I have two factor authentication on and I checked if my billing details are correct.

**Comment by Leokratis** [comment]
If you think that the issue is the camera you should know that you are able to upload files. If you don't see that option, you could try using a computer without a webcam. However, there are more reasons why your application might be rejected. Check if you are sure you fulfill all the other requirements including completing your github profile setup, 2FA, adding billing information and adding your full name as it appears on your (academic) ID.

**Reply by sarthak263** [reply]
Yeah I have completed everything but I don't have a device without a webcam and I don't see any button to upload files. I have 2fa, billing information is completed I believe it's because the picture quality is blurry even it's hard for me to read. Is there any other way?

**Reply by Leokratis** [reply]
I have no idea. Worst case scenario, run a vm with no camera (this isn't a solution I just can't think of anything else)

**Comment by Sibam-Paul** [comment]
I have also applied for GitHub Developer Student pack. I have submitted my College ID card through my laptop front camera and along that there was another section to be uploaded why are u away from the school campus. In that case I uploaded my this semester payment info stating that I'm a day scholar. And the ID card was little blur yet readable.

**Reply by sarthak263** [reply]
It's an online program where we can do masters and work at the same time. John Hopkins online Master of Science in AI program.

**Comment by ianshulx** [comment]
I first applied in my 1st year and faced the same issue despite re-verifying twice. Then, I tried using my phone to capture clearer images, which helped. Also, make sure to link your student email with your GitHub account and upload an ID or document that shows your session's start and end dates, if you don't have a valid proof you can try screen capture of your student erp, result, academic profile etc.

**Comment by DhareshP** [comment]
use your student email id, i.e with @jhu.edu.in

**Reply by Sibam-Paul** [reply]
Yeah after submitting the request just added my college Email will It work ??

**Reply by DhareshP** [reply]
if you verified it using github via otp first/ or just add it as primary or sec email , then you submit the req for student developer pack, just verify it

**Reply by sarthak263** [reply]
Yes my primary email address is jhu and still it doesn't work. It sucks that there is no other way
